app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -26,14 +26,12 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProviders
 import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
 import com.droidhats.campuscompass.MainActivity
 import com.droidhats.campuscompass.R
 import com.droidhats.campuscompass.adapters.SearchAdapter
 import com.droidhats.campuscompass.helpers.Subject
-import com.droidhats.campuscompass.models.Building
-import com.droidhats.campuscompass.models.CalendarEvent
-import com.droidhats.campuscompass.models.GooglePlace
-import com.droidhats.campuscompass.models.NavigationRoute
+import com.droidhats.campuscompass.models.*
```

Comment:
Please don't use * when importing packages.
We have discussed it's better to specify the required components.